app/routes/applications.js

```diff
@@ -114,19 +114,37 @@ module.exports = router => {
     })
   })
 
-  router.get('/applications/:id/school-placement', (req, res) => {
+  router.get('/applications/:id/school-placement(-second)?', (req, res) => {
+    console.log(req.params);
     const { id } = req.params
+    var priority = req.params[0]
```

Comment:
`var priority = req.params[0]` should be `let priority = req.params[0]` i.e., replace `var` with `let`.